modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -348,7 +356,7 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 		pixel_y = 10
 		update_icons()
 		throw_at(A, MAX_K9_LEAP_DIST, 1, spin=0, diagonals_first = 1)
-		cell.use(500) //Doubled the energy consumption
+		cell.use(1000) //Same as stunbaton

```

Comment:
I'd keep this where it's at, since your changes already require multiple pounces rather than just one. Increasing this to 1000 results in a net power consumption increase of at least 4x when downing a single target; realistically, this number is bound to be even higher, since the target is now capable of potentially dodging subsequent pounces. 

In short, I see no reason to increase the energy usage. If anything, it should be decreased. It's not like stun batons use charge every time you miss.

Reply:
The secborg's stun baton uses 1000 per hit and requires 4 batons to stuncrit. This will make it similar as

Reply:
Why does it need to be similar when they are not the same thing? also 4x. after about 9 pounces i'd have a dead battery if I am calculating this right. regardless of most other things consuming power.

I don't like this.

Reply:
That's a fair point. I suppose we'll need to see how these changes compare to the stunbaton in actual gameplay.

Reply:
A secborg also has a dead battery by 9 batons. It is very much the same thing. If it wasn't then it wouldn't stun the same way a baton does. It's just different in that it has a range and used to do a full stamcrit before.

Reply:
yeah but you can't miss with a stun baton and still have it use power, it'll only use power when it successfully lands a hit

Reply:
That's fair, i'll adjust power usage down to 750. I'll also lower the spool up time and lower the knockout time if they miss.